drake/systems/plants/KinematicsCache.h

```diff
@@ -121,6 +126,44 @@ class KinematicsCache
     }
   }
 
+  template <typename Derived>
+  Eigen::Matrix<typename Derived::Scalar, Derived::RowsAtCompileTime, Eigen::Dynamic> transformVelocityMappingToPositionDotMapping(const Eigen::MatrixBase<Derived>& mat) const
```

Comment:
I don't understand. `mat` is supposed to be the matrix of a linear map 
![image](https://cloud.githubusercontent.com/assets/2585880/12017413/5bd93b72-ad56-11e5-9565-77b9bf8d072d.png)

The output is supposed to be the matrix of a linear map 
![image](https://cloud.githubusercontent.com/assets/2585880/12017408/507bda5a-ad56-11e5-8754-4dd522aeb07c.png)

So `mat` is in ![image](https://cloud.githubusercontent.com/assets/2585880/12017431/a19310a2-ad56-11e5-8d32-6dfdc1305477.png) and the output is in ![image](https://cloud.githubusercontent.com/assets/2585880/12017422/8589da1c-ad56-11e5-85fa-9797409406a9.png)

What row vectors are your referring to?